### PR TITLE
Adding Solitaire game

### LINKED ISSUE #4564: [New game]:  Solitaire
### 🎮 Game Request

Game logic and basic description:
Solitaire is a classic single-player card game where the objective is to move all cards to four foundations in ascending order (Ace to King) (52 cards) and separated by suit. The aim is to arrange the cards in some systematic order or, in a few cases, to pair them off to discard them.

### Point down the features

Game points:
The game is won when all cards have been moved to the four foundation piles, with only a single suit in each pile, and all cards being played in ascending sequence. The game is over if there are no more moves that can be made.

### Select program in which you are contributing

_No response_

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by nikki-05**
Created a new game page for Solitaire with the simplest design. ->Implemented game logic for Solitaire, including shuffle, deal, and move functionalities. ->made the game easy and simple
->Tested the Solitaire game for various browsers

## PR Description 📜

Integrated Solitaire game into the existing game website.
Created a new game page for Solitaire with a simple design.
Implemented game logic for Solitaire, including shuffle, deal, and move functionalities.
Added card movements and interactions.
Instructions and rules for playing Solitaire are included on the game page.
Tested the Solitaire game for various browsers and screen sizes to ensure compatibility and responsiveness.
Fixed minor bugs related to card placement and game reset functionality.
will be fixing more issues related to the game
-Added the screenshot  of the game image in the assets folder 
-Made necessary changes to the CSS styling and enhancements
-Added link to the game in the main README.md
Fixes #4564 

<hr>
 
## Mark the task you have completed ✅

- [ x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x ] I have performed a self-review of my own code or work.
- [ x] I have commented my code, particularly in hard-to-understand areas..
- [x ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x ] I have added entries for my game in main README.md
- [x ] I have added README.md in my folder 
- [ x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @nikki-05 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by nikki-05** [comment]
> Thank you @nikki-05 ,for creating the PR and contributing to our GameZone 💗 Review team will review the PR and will reach out to you soon! 😇 Make sure that you have marked all the tasks that you are done with ✅. Thank you for your patience! 😀

hello sir, how do I tick mark ?

**Comment by nikki-05** [comment]
hii @kunjgit may I know any update regarding the PR

**Review by ishita-43 — CHANGES_REQUESTED** [review]
Do the following changes 
add your game image in asset folder of the gamezone follow naming convention while naming the image add your game entry to the main readme.md file of the gammezone follow format (refer other entries)
also in your pr description mark anychanges that applied to your pr example part of program

File: assets/images/Screenshot 2024-07-02 185338.png
Comment (ishita-43) [inline assets/images/Screenshot 2024-07-02 185338.png:1]: rename as per naming convention for images should be same as game folder name

**Review by ishita-43 — CHANGES_REQUESTED** [review]
also add your game entry into the main readme.md file of the repo refer other entries for the format

File: assets/images/Screenshot 2024-07-02 185338.png
Comment (nikki-05) [reply assets/images/Screenshot 2024-07-02 185338.png:1]: hii can you check please if it is done

Comment (ishita-43) [reply assets/images/Screenshot 2024-07-02 185338.png:1]: its still not done you need to update readme.md file of the GameZone repo

**Comment by kunjgit** [comment]
Thank you  @nikki-05 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request closed without merging**